lua/orgmode/parser/logbook.lua

```diff
@@ -134,7 +134,8 @@ function Logbook:recalculate_estimate(line)
 end
 
 ---@param lines string
----@param dates string
+---@param node unknown
```

Comment:
You can type this as `userdata`, since tree-sitter nodes are of that type. Same thing for the `_parse_clocks` params.

Reply:
I've update this as well